src/storeReducers/productsReducer.js

```diff
@@ -1,12 +1,15 @@
 import axios from 'axios';
 
+import {
+  EDIT_PRODUCT,
+  DELETE_PRODUCT,
+  SET_PRODUCTS,
+  SET_DETAIL_PRODUCT,
+} from '../actions/actionTypes';
+
 // This is just to hook in some methods into the main store
 import store from '../store';
```

Comment:
I don't think we need store here any longer.

Reply:
Ill remove all the extra stuff when I am sure nothing is broken during or after the conversion...